### ISSUE TITLE
[Linux] CLI tries to delete /usr/local/bin on uninstall

### ISSUE THREAD

**Issue by ffflorian** [crowdin/crowdin-cli#90]
When I removed the crowdin-cli (v2.0.24) via `sudo apt remove crowdin` I received the following warning:

```
dpkg: warning: while removing crowdin, directory '/usr/local/bin' not empty so not removed
```

It's good that dpkg did not delete the folder and this just appears as a warning. But a package should never delete the `/usr/local/bin` folder completely.

**Comment by v3ctorman** [comment]
Hi, @ffflorian 
Thanks for reporting it. I'm passing it to our dev team for further investigation

**Comment by marr** [comment]
yikes

**Comment by ffflorian** [comment]
Any update on this?

**Comment by v3ctorman** [comment]
@ffflorian Hi! Yes, devs have added this bug to our backlog for future CLI patch. I can't give exact ETA at this moment, however. We will definitely update you on this issue here when the patch will be deployed

**Comment by rohalskyy** [comment]
Hi @ffflorian  ! I've got some good news on the matter. The issue you've reported has been successfully fixed in the latest 2.0.25 version. Would you please re-install CLI from scratch and check whether it's solved now?
